### DISCUSSION TITLE
Fabric Verion

### DISCUSSION THREAD

**Discussion by Harold-Brooks** [P3pp3rF1y/SophisticatedBackpacks#458] [Feature / Idea Archive]
I love this mod but I am a fabric user myself, and therefore can't use it I would love it if you created a fabric port for this! thanks for reading.
                                         

                                                                                      Sincerely,
                                                                                             Harold

**Comment by Gabitil** [comment]
up, this mod is too useful to not be in fabric, theres nothing like it in fabric.
